downstream.go

```diff
@@ -227,6 +227,7 @@ var permanentDownstreamCaps = map[string]string{
 	"server-time":   "",
 	"setname":       "",
 
+	"draft/metadata-2":        "before-connect,max-keys=0,max-value-bytes=1",
```

Comment:
We're missing `draft/metadata-notify-2` here I think.

Reply:
Will be done in a separate MR:

> If the draft/metadata-notify-2 capability is requested, clients also receive metadata updates for the users they are currently monitoring.

We do not support that yet